annotation_pipeline/populate_labelstudio.py

```diff
@@ -0,0 +1,172 @@
+from huggingface_hub import hf_hub_download
+import pandas as pd
+import subprocess
+import sys
+import os
+import configparser
+
+# Add the directory containing label studio interface tools to sys.path
+script_dir = os.path.abspath('../label_studio_interface/')
```

Comment:
Quick note about doing it this way: This will work if you run the program from within a folder that is a subdirectory of the root project directory, but will fail if someone/something runs the script from a different location such as the root of the project directory. Might be best to use either `sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))` or the `from_root` module to avoid issues like this. Then specify the parent package in the `import` as in `main.py`